src/sentry/eventstore/snuba/backend.py

```diff
@@ -1,10 +1,59 @@
 from __future__ import absolute_import
 
+from datetime import datetime
 
+from sentry.utils import snuba
+from sentry.models import SnubaEvent
 from sentry.eventstore.base import EventStorage
 
+DEFAULT_START = datetime.utcfromtimestamp(0)  # will be clamped to project retention
```

Comment:
I think you have the same problem of https://github.com/getsentry/sentry/pull/14249 with DEFAULT_START and DEFAULT_END being initialized at application start.

Reply:
Thanks